### PR TITLE
[android] updated layouts of special notifications

### PR DESCRIPTION
**Pull request by marcelinkaaa**
Updates to the special notifications appearing above our features. 
On smaller screens the height of the notification was not representative of the one set in layout img.1, that was in part due to the use of 9patch drawable as a background. Also text was done as an image. I updated the layouts to use xml only. img. 2. Height is now preserved on all screens.

### CODE REVIEW

File: android/res/drawable/view_less_results_notification_background.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (gubatron) [inline android/res/drawable/view_less_results_notification_background.xml]: license

File: android/res/layout/view_less_results_notification.xml [hunk 2]
```diff
@@ -18,17 +18,39 @@
...
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@drawable/view_less_results_background_yellow"
-    android:gravity="center_vertical|center_horizontal"
-    android:orientation="horizontal">
-    <ImageView
-        android:id="@+id/view_less_results_notification_image"
+    android:layout_height="wrap_content">
+
+    <LinearLayout
```
Comment (gubatron) [inline android/res/layout/view_less_results_notification.xml]: @aldenml marcelinkaaa says that this nesting was the only way the layout would preserve the height that she wanted.

File: android/res/values/strings.xml [hunk 3]
```diff
@@ -692,4 +692,7 @@
     <string name="or">or</string>
     <string name="include">Include</string>
     <string name="exclude">Exclude</string>
+    <string name="seeing_less_results">Seeing less or no results?</string>
```
Comment (gubatron) [inline android/res/values/strings.xml]: this is a nice plus

Comment (gubatron) [inline android/res/values/strings.xml]: CC: @alejandroarturom , and we're not ready yet, more strings are coming

File: android/res/layout/view_less_results_notification.xml (hunk 2 above)
Comment (aldenml) [inline android/res/layout/view_less_results_notification.xml]: let's see if it is necessary

**Pull request merged**